src/monitoring/index.ts

```diff
@@ -116,8 +112,11 @@ export default class MonitoringService {
    */
   getRedirectionDomains() {
     const SYSTEM_GITHUB_TOKEN = config.get("github.systemToken")
-    const OctokitRetry = Octokit.plugin()
-    const octokitWithRetry = new OctokitRetry({
+    // seems to be a bug in typing, this is a direct
+    // copy paste from the octokit documentation
+    // https://octokit.github.io/rest.js/v20#automatic-retries
+    const OctokitRetry = Octokit.plugin(retry as any)
```

Comment:
not sure if this is required! saw your comment but going to the linked site doesn't seem to suggest this.

![image](https://github.com/isomerpages/isomercms-backend/assets/44049504/5b64d1c3-8ad5-4c55-b86e-9ed9e85c6d76)

this might be due to our `tsconfig` not changing, which [seems](https://github.com/octokit/plugin-retry.js/) to be required by the `retry` library 
![image](https://github.com/isomerpages/isomercms-backend/assets/44049504/ee265a5b-0e3b-4107-993e-c7787dfdbc18)

i think we might want to update our `tsconfig` and check again

Reply:
sadly i get this even after changing 
<img width="1297" alt="Screenshot 2024-06-06 at 3 54 25 PM" src="https://github.com/isomerpages/isomercms-backend/assets/42832651/3de8827d-b434-488b-beb8-32249a0a92dd">
<img width="258" alt="Screenshot 2024-06-06 at 3 55 10 PM" src="https://github.com/isomerpages/isomercms-backend/assets/42832651/6d83bb7e-6a13-450f-b076-51de8fedf1d7">

Reply:
hey @seaerchin, this has since been update by increasing the version number, thanks for the catch